microsoft-edge/devtools-guide-chromium/whats-new/2022/10/devtools-107.md

```diff
@@ -0,0 +1,124 @@
+---
+title: What's new in DevTools (Microsoft Edge 107)
+description: "Automate WebView2 with Playwright. Focus Mode Activity Bar icons have tooltips. New shortcut key for Command Palette. Memory tool loads larger heap snapshots. Links in Application tool render in high contrast mode. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 10/31/2022
+---
+# What's New in DevTools (Microsoft Edge 107)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Automate WebView2 with Playwright
+
+<!-- Subtitle: You can now use Playwright to automate and test web content in the Microsoft Edge WebView2 control. -->
+
+[Playwright](https://playwright.dev) is a testing library that enables reliable end-to-end testing across different browsers and platforms.  Playwright has always supported automating and testing web content in Microsoft Edge, but now Playwright also supports testing the Microsoft Edge WebView2 control.
+
+The Microsoft Edge WebView2 control allows you to embed web technologies (HTML, CSS, and JavaScript) in your native apps.  You can now use Playwright to test your web content running in WebView2.  To get started, see [Playwright's documentation for WebView2](https://playwright.dev/docs/webview2).
+
+![Playwright logo](./devtools-107-images/playwright.png)
```

Comment:
Not sure I get the point of this image. I would recommend just cutting this.

Reply:
removed